types/marked-terminal/index.d.cts

```diff
@@ -0,0 +1,61 @@
+// Reference/dependency missing in chalk
+/// <reference types="node" />
+
+import { CardinalOptions } from "cardinal";
+// @ts-ignore - should use `with { "resolution-mode": "import " }`
+// but typescript-eslint doesn't parse it yet
+import type { ChalkInstance } from "chalk";
```

Comment:
I don’t know how bad everyone feels this is. Do we need to wait for typescript-eslint?